examples/tests/valid/static_members.w

```diff
@@ -0,0 +1,15 @@
+resource Foo {
+  instance_field: num;
+
+  static f: num;
```

Comment:
I _think_ it might be okay for now not to support static fields...

Reply:
I might take you up on this since it'll remove the need to handle static field initialization which is a bit complex.

Reply:
Fuck

Reply:
> I guess we can start with just supporting this in jsii import without supporting the syntax to declare and define the initial value in Wing. No?

Exactly my thoughts. Added a tracking issue and resolving this.